src/java.desktop/share/classes/javax/swing/CellEditor.java

```diff
@@ -85,10 +85,10 @@ public interface CellEditor {
 
     /**
      * Returns true if the editing cell should be selected, false otherwise.
-     * Typically, the return value is true, because is most cases the editing
+     * Typically, the return value is true, because in most cases the editing
      * cell should be selected.  However, it is useful to return false to
-     * keep the selection from changing for some types of edits.
-     * eg. A table that contains a column of check boxes, the user might
+     * keep the selection from changing for some types of edits,
+     * eg. in a table that contains a column of check boxes, the user might
```

Comment:
Most guides recommend to spell _“e.g.”_ with the period after each letter, see [an article at Grammarly](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/). Alternatively, it could be replaced with _“for example”_.

Reply:
yes, it should be "e.g.",